content/docs/components/serving/kfserving.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "KFServing"
+description = "Using KFServing for serving Models"
+weight = 2
++++
+
+## KFServing with Kubeflow
+
+KFServing can be installed with Kubeflow v0.7, and KFServing Kustomize install files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
+
+KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and ONNX. 
+
+KFServing encapsulates the complexity of autoscaling, networking, health checking, and server configuration to bring cutting edge serving features like GPU Autoscaling, Scale to Zero, and Canary Rollouts to your ML deployments. It enables a simple, pluggable, and complete story for production ML Inference Server by providing prediction, pre-processing, post-processing and explainability out of the box.
```

Comment:
s/Inference Server/inference servers

Similar to previous comments, remove the capital letters, and the term should be plural here as it's generic.